server/routes/v1/tokenDiscovery.js

```diff
@@ -114,21 +125,63 @@ async function cacheTokensList() {
   }
 }
 
-if (!timerStarted && process.env.REACT_APP_ENVIRONMENT === 'mainnet') {
-  timerStarted = true;
-  cacheTokensList();
-  setInterval(() => cacheTokensList(), TIME_INTERVAL);
+// Starts the caching process for bigquery
+function startCaching() {
+  // Only run if on mainnet (the tokens page doesn't exist on devnet/testnet)
+  if (process.env.REACT_APP_ENVIRONMENT === 'mainnet') {
```

Comment:
since the body of the function is within the `if` you can remove an indentation by reversing this:

```javascript
function startCaching() {
  if (!process.env.REACT_APP_ENVIRONMENT === 'mainnet') {
    return
  }
  // rest of function
}
```